tests/apollo/test_skvbc_s3_integrity_check.py

```diff
@@ -0,0 +1,181 @@
+# Concord
+#
+# Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the Apache 2.0 license (the "License").
+# You may not use this product except in compliance with the Apache 2.0 License.
+#
+# This product may include a number of subcomponents with separate copyright
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+
+import unittest
+import trio
+import os
+import subprocess
+
+from minio import Minio
+from util.test_base import ApolloTest
+from util import skvbc as kvbc
+from util import eliot_logging as log
+from util.object_store import ObjectStore, start_replica_cmd_prefix
+from util.bft import KEY_FILE_PREFIX, with_trio, with_bft_network
+
+def start_replica_cmd(builddir, replica_id, config):
+    """
+    There are two test s3 config files. The one used here hasn't got s3-prefix parameter
+    set which means that on each RO replica start the prefix is some unique time based
```

Comment:
This was fixed recently: if not provided, prefix will be an empty string

Reply:
fixed.